.github/workflows/cifuzz.yml

```
@@ -54,9 +55,17 @@ jobs:
           fuzz-seconds: 600
           dry-run: false
           sanitizer: ${{ matrix.sanitizer }}
+          output-sarif: true
       - name: Upload Crash
         uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb
         if: failure() && steps.build.outcome == 'success'
         with:
           name: ${{ matrix.sanitizer }}-${{ matrix.architecture }}-artifacts
           path: ./out/artifacts
+      - name: Upload Sarif
+        if: always() && steps.build.outcome == 'success'
+        uses: github/codeql-action/upload-sarif@v2
```

Please pin to SHA `github/codeql-action/upload-sarif` GA - https://github.com/github/codeql-action/releases/tag/codeql-bundle-20230418

```suggestion
        uses: github/codeql-action/upload-sarif@9a866ed4524fc3422c3af1e446dab8efa3503411
```